ccxt-service/Dockerfile

```
@@ -11,12 +11,12 @@ RUN apt-get update && apt-get install -y --no-install-recommends curl ca-certifi
 FROM base AS install
 RUN mkdir -p /temp/dev
 COPY package.json bun.lock /temp/dev/
-RUN cd /temp/dev && bun install --frozen-lockfile
+RUN cd /temp/dev && bun install
```

Hadolint (2.12.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)